README.md

```diff
@@ -301,6 +301,25 @@ In particular:
 
 
 
+#### Github using SSH
+To use terragrunt with SSH github authentication, use the following syntax. Note the `/` after 
+github.com instead of `:` and the uses of `//`. 
+
+```hcl
+terragrunt = {
+  terraform {
+    source = "git::ssh://git@github.com/foo/modules.git//path/to/module?ref=v0.0.1"
+  }
+}
+```
+For private git repositories, replace github.com with the private repository host. 
```

Comment:
I would replace this sentence with "Look up the Git repo for your repository to find the proper format." After all, GitHub has private repos too, and other types of private Git repos may have significantly different URL formats. For example, for private BitBucket repos, you may need to use a URL like `git::ssh://git@my-bitbucket-account.com:7999/team-name/repo-name.git//path/to/module?ref=v0.0.1`.